.env.example

```diff
@@ -1,8 +1,8 @@
 LINKFREE_MONGO_CONNECTION_STRING="mongodb://localhost:27017/linkfree"
-NEXT_PUBLIC_GA_MEASUREMENT_ID=""
+NEXT_PUBLIC_GA_MEASUREMENT_ID="-"
 NEXT_PUBLIC_BASE_URL="http://localhost:3000"
 NODE_ENV="development"
 
-GITHUB_ID=""
-GITHUB_SECRET=""
-NEXTAUTH_SECRET=""
+GITHUB_ID="10c01b64991cf71735d1"
+GITHUB_SECRET="c2a58bbcea3cef1dd3c847eea392e0f2369b39f2"
+NEXTAUTH_SECRET="afsfdsafadsf"
```

Comment:
You should not commit your secrets here, please remove these changes and reset your GitHub OAuth secrets